### PR TITLE
feat: update example address to 0x0

### LINKED ISSUE #549: Update example wallet addresses to `0x0`
If we use `0x0000000000000000000000000000000000000000` then it will clearly signal that the wallet assign command was not used correctly.

1. In every example including on the help menu, use the `0x0000000000000000000000000000000000000000` address
2. In the assign command there should be a specific error for if a user attempts to assign their wallet to `0x0000000000000000000000000000000000000000` 

---

> Can you update all examples to this `0x0000000000000000000000000000000000000000` address so that we can catch this in the assign command easily?

_Originally posted by @pavlovcik in https://github.com/ubiquity/ubiquibot/pull/528#discussion_r1276210334_

### PR DESCRIPTION
**Pull request by me505**
Resolves #549

### CODE REVIEW

**Review by pavlovcik — APPROVED** [review]

**Comment by pavlovcik** [comment]
I'm assuming you checked for every example address across all commands?

**Review by pavlovcik — APPROVED** [review]
I fixed the merge conflict for you.

**Comment by me505** [comment]
> I fixed the merge conflict for you.

thanks

File: src/handlers/comment/handlers/wallet.ts [hunk 1]
```diff
@@ -69,6 +69,10 @@ export const registerWallet = async (body: string) => {
   }
 
   if (address) {
+    if (address == "0x0000000000000000000000000000000000000000"){
```
Comment (0xcodercrane) [inline src/handlers/comment/handlers/wallet.ts:72]: Avoid using the hardcoded `0x00..0` address. there should be a AddressZero constant in ethers library.

**Review by 0xcodercrane — CHANGES_REQUESTED** [review]

**Comment by me505** [comment]
weird, [netlify](https://app.netlify.com/sites/ubiquibot-staging/deploys/64c34834731ecf0008a20ee1) checks seems to have passed

**Review by pavlovcik — APPROVED** [review]

**Review by 0xcodercrane — APPROVED** [review]

**Pull request merged**